go/consensus/tendermint/abci/context.go

```diff
@@ -77,12 +77,14 @@ type Context struct {
 }
 
 // NewMockContext creates a new mock context for use in tests.
-func NewMockContext(mode ContextMode, now time.Time) *Context {
+func NewMockContext(mode ContextMode, now time.Time, state *iavl.MutableTree) *Context {
```

Comment:
We should remove this now that we have mock application state (I actually removed it in #2674). Can you use `NewMockApplicationState` in tests instead as it already handles creating a new in-memory tree for you?